web-ui/src/main/resources/ui/app/Visualization.js

```diff
@@ -44,8 +44,6 @@
  */
  define(function(require) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["require" is defined but never used](https://www.codacy.com/app/frank_7/spawncamping-dds/file/2026401273/issues/source?bid=3046234&fileBranchId=3126677#l45)